Install error on 32bit linux

I have naabu installation error on a 32bit pc. My guess is 32/64 bit difference. Is there a solution?

Error log
```
$ go get -v github.com/projectdiscovery/naabu/cmd/naabu
github.com/projectdiscovery/naabu/pkg/scan
# github.com/projectdiscovery/naabu/pkg/scan
../../go/src/github.com/projectdiscovery/naabu/pkg/scan/scan.go:143:36: constant 8999999999 overflows int
```

ENV
```
- Ubuntu 18.04.4 LTS (kernel 4.15.0-88-generic)
- 32bit computer
- installed libpcap-dev
```

--- a/pkg/scan/scan.go
+++ b/pkg/scan/scan.go
@@ -3,6 +3,7 @@ package scan
 import (
 	"fmt"
 	"io"
+	"math"
 	"math/rand"
 	"net"
 	"strings"
@@ -140,7 +141,7 @@ func (s *Scanner) Scan(wordlist map[int]struct{}) (map[int]struct{}, error) {
 		OptionLength: 4,
 		OptionData:   []byte{0x12, 0x34},
 	}
-	randSeq := 1000000000 + rand.Intn(8999999999)
+	randSeq := 1000000000 + rand.Intn(math.MaxInt32)
 
 	tcp := layers.TCP{
 		SrcPort: layers.TCPPort(rawPort),

